<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>IFileEncryptionPasswordDialog</class>
  <widget class="QDialog" name="IFileEncryptionPasswordDialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>419</width>
        <height>193</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>File Encryption Password</string>
    </property>
    <property name="sizeGripEnabled">
      <bool>true</bool>
    </property>
    <property name="modal">
      <bool>true</bool>
    </property>
    <widget class="QLabel" name="textLabel1">
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>20</y>
          <width>360</width>
          <height>20</height>
        </rect>
      </property>
      <property name="text">
        <string>Please enter password for:</string>
      </property>
      <property name="wordWrap">
        <bool>false</bool>
      </property>
    </widget>
    <widget class="QLineEdit" name="leFilePassword">
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>90</y>
          <width>350</width>
          <height>31</height>
        </rect>
      </property>
      <property name="echoMode">
        <enum>QLineEdit::Password</enum>
      </property>
    </widget>
    <widget class="QWidget" name="Layout1">
      <property name="geometry">
        <rect>
          <x>80</x>
          <y>140</y>
          <width>300</width>
          <height>32</height>
        </rect>
      </property>
      <layout class="QHBoxLayout">
        <property name="margin">
          <number>0</number>
        </property>
        <property name="spacing">
          <number>6</number>
        </property>
        <item>
          <spacer name="Horizontal Spacing2">
            <property name="sizeHint">
              <size>
                <width>20</width>
                <height>20</height>
              </size>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="orientation">
              <enum>Qt::Horizontal</enum>
            </property>
          </spacer>
        </item>
        <item>
          <widget class="QPushButton" name="buttonOk">
            <property name="text">
              <string>&amp;OK</string>
            </property>
            <property name="shortcut">
              <string/>
            </property>
            <property name="autoDefault">
              <bool>true</bool>
            </property>
            <property name="default">
              <bool>true</bool>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="buttonCancel">
            <property name="text">
              <string>&amp;Cancel</string>
            </property>
            <property name="shortcut">
              <string/>
            </property>
            <property name="autoDefault">
              <bool>true</bool>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
    <widget class="QLabel" name="tlFileName">
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>50</y>
          <width>360</width>
          <height>20</height>
        </rect>
      </property>
      <property name="text">
        <string>filepath</string>
      </property>
      <property name="wordWrap">
        <bool>false</bool>
      </property>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <connections>
    <connection>
      <sender>buttonOk</sender>
      <signal>clicked()</signal>
      <receiver>IFileEncryptionPasswordDialog</receiver>
      <slot>accept()</slot>
    </connection>
    <connection>
      <sender>buttonCancel</sender>
      <signal>clicked()</signal>
      <receiver>IFileEncryptionPasswordDialog</receiver>
      <slot>reject()</slot>
    </connection>
  </connections>
</ui>
